sugar.go

```diff
@@ -422,7 +422,7 @@ func (k *Keyboard) InsertText(text string) {
 // Describe returns the element info
 // Returned json: https://chromedevtools.github.io/devtools-protocol/tot/DOM#type-Node
 func (el *Element) Describe() *proto.DOMNode {
-	node, err := el.DescribeE()
+	node, err := el.DescribeE(-1, true)
```

Comment:
By default, I feel it's better to use `(1, false)`, most time we just use it to get the current node info, not the tree. Or can you give a strong reason why we should return huge data that may impact performance by default?